packages/cascara/src/ui/Table/index.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import { TableContextProvider } from './context/TableContext';
+import Table from './Table';
+
+export default ({ data, dataConfig, onAction }) => (
+  <TableContextProvider data={data} dataConfig={dataConfig} onAction={onAction}>
+    <Table />
+  </TableContextProvider>
+);
```

Comment:
I see... we are using the index.js to set the parameters.

This might present some issues for us in terms of how the exporting and naming of the components work. For instance, our doc typing I do not think will work with this if we define `index.js` as the file to run typing on. The docgen I think will also fail because I do not think exporting default from an index file will apply a component name. 

Maybe this will be okay if we define a component name? I think the problem with that is we are defining the index as Table and then we also have a component called Table. This is likely to be a confusing pattern for devs. 

- [ ] Let's talk about this together and figure out what the pros and cons of this pattern are.

Reply:
yup this is not a good pattern, I didn't want to spend time on fixing it because it was just a fixture.
Let me finish the convertion to React components and fix this, thanks for the catch!

Reply:
The change has been applied, Table is the context container and the index only exports table.